DESCRIPTION

```diff
@@ -1,7 +1,7 @@
 Package: Rdisop
 Title: Decomposition of Isotopic Patterns
-Version: 1.67.5
-Date: 2025-01-21
+Version: 1.67.6
```

Comment:
Should be 1.69.1, I forgot to push to github devel after BioC release in April. Will fix here.